buildSrc/src/main/kotlin/com/datadog/gradle/plugin/jsonschema/PokoSerializerGenerator.kt

```diff
@@ -45,7 +45,7 @@ class PokoSerializerGenerator {
 
         when (additionalProperties) {
             is TypeDefinition.Primitive -> funBuilder.addStatement("json.addProperty(k, v)")
-            is TypeDefinition.Class,
+            is TypeDefinition.Class -> funBuilder.addStatement("json.add(k,v.toJsonElement())")
```

Comment:
you need a space after a comma

Reply:
Also it'd be better to use the same constant for the method name (a.k.a `TO_JSON_ELEMENT_EXTENSION_FUNCTION`)